### ISSUE TITLE
Add a menu bar and move some functionality to it

### ISSUE THREAD

**Issue by yamsgithub** [VIDA-NYU/domain_discovery_tool_deprecated#9]
- [x] Add a menu bar on the top and move following functionality to it to reduce the space of the interface:
  1. Domains list
  2. Add new domain
  3. ~~Clustering methods~~
  4. Model building
- [x] The menu bar should be fixed while scrolling. 
- [x] It should display the name of the currently active domain selected.
- [x] Display the name "Domain Discovery Tool" and logo on the left corner.

**Comment by britthard** [comment]
@yamsgithub will I be working from the hand-written mockups that @aecio made? Have there been any changes or decisions made for the design?

**Comment by yamsgithub** [comment]
I think that should be fine. There have been no other design decisions made. I believe Katrina has updated you on the time frame. Since we want to make the Tool review on Sep 8th it would be great if you could have something for us on Mon next week.

**Comment by britthard** [comment]
Sorry, do you mean Monday September 7th? Or August 31st?

**Comment by yamsgithub** [comment]
Sorry...my bad...I meant August 31st.

**Comment by britthard** [comment]
The last time I talked to Katrina about it she wanted to get the work over the course of next week. I'm not sure that I will be able to close this ticket by Monday. @kriehl can you weigh in on this?

**Comment by yamsgithub** [comment]
@brittainhard you don't have to close the ticket. But would be good to have something ready by then so we can iterate over it if necessary. But this has to be closed no later than Sep 2nd as I will need some time for full integration with the backend and testing before we put it in front of the surrogate user group review.

**Comment by britthard** [comment]
Is "Tool review" and the surrogate user group review the same thing?

**Comment by yamsgithub** [comment]
Yes.

**Comment by britthard** [comment]
I was planning on closing this whole issue as a PR. If theres some piece of this ticket that you wanted done first, so that it could be done by Monday, it would be useful to know what that piece is so I can focus my time on it, instead of focusing on the ticket generally. It would also be useful to have that as a separate issue.

As is, this ticket will not be able to be closed by Monday. I still have to dedicate some time to understanding the codebase enough that I can make changes to it without disrupting anything.

Maybe it would be best to start with a part of the menu you want done first? Maybe I can add the menu first, then add a single item to the menu, and follow with the rest of the items.

**Comment by yamsgithub** [comment]
Yes. You could add the menu and starry by first moving the add domain part to it. Once that is done we can move other things to the menu. Does that make sense ?

**Comment by britthard** [comment]
Yes that's fine. I can't as of now give you an assessment as to how long that will take. I'll let you know when I know.

**Comment by britthard** [comment]
@yamsgithub I think I can get the first part of this issue ready to be merged before lunch on tuesday at the latest

**Comment by yamsgithub** [comment]
@brittainhard that sounds good. Thanks!

**Comment by britthard** [comment]
Hey @yamsgithub @aecio I need to include some third-party JS libraries to implement some of the interactions from the mockups. The first of these would be bootstrap.js, which I would need for dropdown menus. I might also need a library for proper styling on the select crawler element. Any thoughts on this?

**Comment by yamsgithub** [comment]
We already include bootstrap I think. If not feel free to add any libraries you need.

**Comment by britthard** [comment]
@yamsgithub I noticed that you crossed out the last two items "Clustering methods" and "Model building".

Do you still want me to add these things to the menu?

**Comment by yamsgithub** [comment]
Actually we decided not to move the clustering methods. But we still want
to move the Model Building to the menu.

**Comment by britthard** [comment]
Awesome. I'll have a PR for that up soon.

**Comment by aecio** [comment]
Done in pull requests #28, #32 and #33.
